snap-server.cabal

```diff
@@ -91,7 +91,7 @@ Library
     bytestring                          >= 0.9.1    && < 0.11,
     bytestring-builder                  >= 0.10.4   && < 0.11,
     case-insensitive                    >= 1.1      && < 1.3,
-    clock                               >= 0.4.1    && < 0.6,
+    clock                               >= 0.4.1    && < 0.7,
```

Comment:
@imalsogreg wow, these are very bad news, as stackage 4.1 has clock included into itself. I'll create a ticket to discuss this!